src/geometry_aux.cpp

```diff
@@ -23,6 +23,25 @@
 
 namespace openmc {
 
+namespace model {
+  std::map<int32_t, std::map<int32_t, int32_t>> universe_cell_counts;
+  std::map<int32_t, int32_t> universe_level_counts;
+} // namespace model
+
+void update_universe_cell_count(int32_t a, int32_t b) {
```

Comment:
It's not obvious what the purpose of this function is in isolation. Can you add a comment above explaining it?